pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -25,25 +25,43 @@ public PythonVersion(String version) {
         this.version = version;
     }
 
+    private String getVersionPart(int position) {
+        String[] parts = this.version.split("\\.");
+        return parts[position];
+    }
+
     /**
      * @return The exact version of Python this project uses, such as '2.7.11'.
      */
     public String getPythonVersion() {
-        return version;
+        return this.version;
     }
 
     /**
      * @return The short version of Python this project uses, such as '2.7'.
      */
     public String getPythonMajorMinor() {
-        String[] split = version.split("\\.");
-        return String.format("%s.%s", split[0], split[1]);
+        return String.format("%s.%s", getVersionPart(0), getVersionPart(1));
```

Comment:
Hmm. Two method calls with two splits instead of one split. I'm inclined to leave this method unchanged. It's working just fine. Although, I can see why you would want the consistency and isolation of this handling in the `getVersionPart` only.